kuma/javascript/src/utils.js

```diff
@@ -0,0 +1,16 @@
+// Helper to get cookie values
+export function getCookie(name) {
```

Comment:
Add JSDoc style comment. The name is self-evident but, would be worth doing so we can clearly indicate what type `name` is expected to be, and what is returned.

```
/**
 * What does it do?
 * @params {type} name - ...
 * @returns What does it return?
 */
```